File: app/src/androidTest/java/com/example/task2/models/benchmarks/FakeMapBenchmark.java
package com.example.task2.models.benchmarks;

import static com.example.task2.models.benchmarks.Constants.RESULT_MAP;
import static com.example.task2.models.benchmarks.Constants.SLEEP_TIME_FOR_MODELS;

public class FakeMapBenchmark extends MapBenchmark implements Benchmark {

    @Override
    public long measureTime(CellOperation item, int number) {
        try {
            Thread.sleep(SLEEP_TIME_FOR_MODELS);
        } catch (InterruptedException e) {
            e.printStackTrace();
        }
        return RESULT_MAP;
    }
}

File: app/src/androidTest/java/com/example/task2/models/benchmarks/FakeMapBenchmark.kt
package com.example.task2.models.benchmarks

import com.example.task2.Constants

class FakeMapBenchmark : MapBenchmark(), Benchmark {
    override fun measureTime(item: CellOperation, number: Int): Long {
        try {
            Thread.sleep(Constants.SLEEP_TIME_FOR_MODELS)
        } catch (e: InterruptedException) {
            e.printStackTrace()
        }
        return Constants.RESULT_MAP
    }
}